### ISSUE TITLE
examples/core/parallelize.js fails with 2 workers

### ISSUE THREAD

**Issue by mvertes** [skale-me/skale-engine#2]
To reproduce, in terminal 1:
```
$ cd skale-engine
$ ./bin/server -l 2
```

In terminal 2:
```
$ cd skale-engine
$ ./examples/core/parallelize.js
[ 1, 2, 3, 4 ]

assert.js:89
  throw new assert.AssertionError({
  ^
AssertionError: false == true
    at Console.assert (console.js:94:23)
    at /Users/marc/github/skale-engine/examples/core/parallelize.js:9:10
    at /Users/marc/github/skale-engine/node_modules/stream-to-array/index.js:54:9
    at _combinedTickCallback (node.js:376:9)
    at process._tickCallback (node.js:407:11)
 ```

The output should be `[ 1, 2, 3, 4, 5 ]`

**Comment by mvertes** [comment]
The example works correctly with any other number of workers.

**Comment by mvertes** [comment]
fixed by commit 3d0ac334 (which points to a wrong issue, sorry)
